Field-Robotics-Lab/dave
# Update URDF xacros

Fix #200 

To reproduce the problem on `master`: 
1. Upgrade to the latest `xacro` package using the command `sudo apt install --only-upgrade ros-noetic-xacro`. You can then run `dpkg -l | grep ros-noetic-xacro` to view the version of the `xacro` binary. It should be greater than `1.14.10`. 
2. Source `noetic` and build the workspace using `catkin build` or `catkin_make`
3. Source the workspace and run `roslaunch dave_demo_launch dave_demo.launch`. This should lead to the error described in the linked ticket.

To test the fix:
1. Checkout to this branch
2. Build the workspace
3. Source the workspace and launch the demo. The simulation should launch successfully.
